AWSSDKSwiftCLI/Sources/AWSCLIUtils/Error.swift

```diff
@@ -7,11 +7,11 @@
 
 import Foundation
 
-struct Error: LocalizedError {
-    var message: String
-    var errorDescription: String? { message }
-    
-    init(_ message: String) {
+public struct Error: LocalizedError {
```

Comment:
As you will see below, many types like this one have been moved into AWSCLIUtils and been marked public, so that they can be shared between CLI utilities.